packages/angular_devkit/build_angular/src/angular-cli-files/plugins/index-html-webpack-plugin.ts

```diff
@@ -67,41 +68,38 @@ export class IndexHtmlWebpackPlugin {
 
       const loadOutputFile = (name: string) => compilation.assets[name].source();
 
-      // Get all files for selected entrypoints
-      const unfilteredSortedFiles: string[] = [];
+      // Get all files for script references
+      const files = getEmittedFiles(compilation);
```

Comment:
This is not meant to be used like this. the `getEmittedFiles` is not intended to be part of the public api.

Reply:
If you have access to a webpack compilation, you have the chunk of data that you need, the emittedFiles, will return way more files then you need.